### PR TITLE
Support Moment-Timezone & Datasets

### PR DESCRIPTION
**Pull request by hhff**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,11 +11,28 @@ module.exports = {
...
+        case 'all':
+          timezonePath += 'builds/moment-timezone-with-data';
+          break;
+        case '2010-2020':
+          timezonePath += 'builds/moment-timezone-with-data-2010-2020';
+          break;
+        default:
+          timezonePath += 'moment-timezone';
+          break;
+      }
```
Comment (stefanpenner) [inline index.js]: we should likely  error if an invalid `includedTimezone` is provided

**Comment by stefanpenner** [comment]
left one comment

File: index.js (hunk 1 above)
Comment (hhff) [inline index.js]: can do!

**Comment by hhff** [comment]
actioned & rebased @stefanpenner

**Comment by stefanpenner** [comment]
```
± % npm publish                                                                                                                                                               !17927
+ ember-moment@1.2.1
```

**Comment by hhff** [comment]
the mannnn

**Comment by jasonmit** [comment]
Thanks :+1:

**Comment by jasonmit** [comment]
This has changed a bit in 2.0.0, functionality remains the same but the configuration is different and is now handled inside of the shim instead of in ember-moment.

**Pull request merged**